app/src/main/java/com/example/concordia_campus_guide/Adapters/FloorPickerAdapter.java

```diff
@@ -51,13 +52,23 @@ public View getView(final int position, final View view, ViewGroup viewGroup) {
             button.setOnClickListener(new View.OnClickListener() {
                 @Override
                 public void onClick(View v) {
```

Comment:
The only comment I have is the usage of "v" as a parameter name. It is preferred to use a clear parameter names such "view"